mongodb_exporter.go

```diff
@@ -77,6 +83,8 @@ func main() {
 	kingpin.Version(version.Print(program))
 	kingpin.Parse()
 
+	mongod.InitOpLatenciesMetrics(*latencyHistogramMinF, *latencyHistogramStepF, *latencyHistogramCountF)
```

Comment:
You shouldn't use mongod. We use `mongodb_collector.go` as entrypoint, so it will be better if you discover how it used and integrate a flags and initialization inside `mongodb_collector`. See: https://github.com/percona/mongodb_exporter/blob/master/collector/mongodb_collector.go#L273
https://github.com/percona/mongodb_exporter/blob/master/collector/mongod/server_status.go#L67

Reply:
Thank you for reviewing.  The project mongodb_exporter typically has prometheus metrics as global variables which initialize before command line arguments are parsed.

I don't fully follow how they should be initialized using the arguments and following the design pattern of mongodb_exporter.  Could you clarify?

For example I could add the command arguments to MongodbCollectorOpts structure, and add call to mongod.InitOpLatenciesMetrics from NewMongodbCollector()?

Reply:
> For example I could add the command arguments to MongodbCollectorOpts structure, and add call to mongod.InitOpLatenciesMetrics from NewMongodbCollector()?

Yes, it would be great if you do this, and remove `mongod` import from `main.go` because we use it through collector package. Also sign an CLA please and run `go test -update` before commiting. It replace golden file https://github.com/percona/mongodb_exporter/blob/master/testdata/mongodb_exporter.testFlagHelp.golden which is used for testing binary params.

Reply:
Great 👍